File: 024-spark-test/src/main/java/com/demien/sparktest/util/JsonUtil.java
package com.demien.sparktest.util;

import com.google.gson.Gson;

public class JsonUtil {
    public static String toJson(Object object) {
        return new Gson().toJson(object);
    }

    public static Object toObject(String json, Class<?> cl) {
        return new Gson().fromJson(json, cl);
    }
}

File: 047-money-transfer/src/main/scala/com/demien/mtransfer/rest/JsonUtil.scala
package com.demien.mtransfer.rest

import com.google.gson.{Gson, GsonBuilder}


object JsonUtil {

  val gson: Gson = new GsonBuilder()
    .setLenient()
    .create();

  def toJson(entity: Any): String = gson.toJson(entity)

  def fromJson[T](json: String, cl: Class[_]): T = gson.fromJson(json, cl)


}